templates/flutter/lib/package.dart.twig

```diff
@@ -1,7 +1,24 @@
+library {{ language.params.packageName }};
+
+
+import 'dart:io';
+import 'package:universal_html/html.dart' as html;
+import 'package:dio/dio.dart';
+import 'package:meta/meta.dart';
+import 'package:flutter/foundation.dart';
+import 'package:flutter_web_auth/flutter_web_auth.dart';
+import 'package:dio/adapter.dart';
+import 'package:dio_cookie_manager/dio_cookie_manager.dart';
+import 'package:cookie_jar/cookie_jar.dart';
+import 'package:path_provider/path_provider.dart';
+import 'package:package_info_plus/package_info_plus.dart';
+
```

Comment:
Extra space, or is this a Dart convention?

Reply:
signle space is convention, more than one, that's extra, i'll deal with